packages/@react-spectrum/dropzone/test/DropZone.test.js

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {act, fireEvent, render} from '@react-spectrum/test-utils';
+import {Content} from '@react-spectrum/view';
+import {DataTransfer, DataTransferItem, DragEvent} from '@react-aria/dnd/test/mocks';
+import {Draggable} from '@react-aria/dnd/test/examples';
+import {DropZone} from '../';
+import {FileTrigger} from 'react-aria-components';
+import {Heading} from '@react-spectrum/text';
+import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
+import {Link} from '@react-spectrum/link';
+import React from 'react';
+
+describe('DropZone', () => {
+  it('should render a dropzone', () => {
+    let {getByTestId} = render(
+      <DropZone data-testid="foo">
+        <IllustratedMessage>
+          <Heading>No files</Heading>
+          <Content>
+            <FileTrigger>
+              <Link>Select a file</Link>from your computer
+            </FileTrigger>
+          </Content>
+        </IllustratedMessage>
+      </DropZone>
+    );
+    let dropzone = getByTestId('foo');
+    expect(dropzone).toHaveAttribute('class', 'spectrum-Dropzone');
+  });
+
+  it('should render an illustrated message as a child', () => {
+    let {getByTestId} = render(
+      <DropZone>
+        <IllustratedMessage>
+          <Heading data-testid="foo">No files</Heading>
+          <Content data-testid="bar">
+            <FileTrigger>
+              <Link>Select a file</Link>from your computer
+            </FileTrigger>
+          </Content>
+        </IllustratedMessage>
+      </DropZone>
+    );
+
+    let heading = getByTestId('foo');
+    let content = getByTestId('bar');
+    expect(heading).toHaveAttribute('class', 'spectrum-IllustratedMessage-heading');
+    expect(content).toHaveAttribute('class', 'spectrum-IllustratedMessage-description');
+  });
+
+  it('should attach a ref to the dropzone if provided as a prop', () => {
```

Comment:
this should be covered in the suggestion for the first test in this file if it's important

could be argued it should be placed in the RAC test file instead if it's important, in which case we wouldn't need it here